hs628_tools/normalization_tool.py

```diff
@@ -37,7 +35,6 @@ def get_input(self, percent_diff):
         else:
```

Comment:
Really like your idea! And your code is well structured and you explain it clearly enough to understand. 
For the cutoff = 0.3, can I ask where it's come from and why?

Reply:
Cutoff is based on analyzing the mean ratios right now. Mean ratio is actually equal to "minimum mean divided by maximum mean, multiplied by 100". Based on the analysis of values found, closer to 0 will mean that the difference in mean values is really large, and we want to avoid such a scenario. To narrow down to the value of 0.3 as the cutoff, I checked different datasets, and used different cutoffs, and after all the analysis, came up with this value. But I definitely know that depending on dataset, the optimum value of cut-off may change, and hence I have also left this as a user choice, where user has the ability to modify the default cut-off.